cheta/derived/comps.py

```diff
@@ -554,8 +554,28 @@ def get_msid_attrs(self, tstart, tstop, msid, msid_args):
 
 
 @functools.lru_cache(maxsize=1)
-def get_roll_pitch_tlm(start, stop):
-    from .. import fetch
+def get_roll_pitch_tlm_safe_table(start, stop):
+    """Get telemetry values to compute pitch and roll in safe mode"""
+    from cheta.utils import get_telem_table
+
+    msids = ["6sares1", "6sares2", "6sunsa1", "6sunsa2", "6sunsa3"]
+    dat = get_telem_table(msids, start, stop)
+
+    # Filter out of range values. This happens just at the beginning of safe mode.
+    bads = np.zeros(len(dat), dtype=bool)
+    for msid in msids:
+        x0, x1 = 0, 180 if ("6sares" in msid) else -1, 1
+        bads |= (dat[msid].vals < x0) | (dat[msid].vals > x1)
+    if np.any(bads):
+        dat = dat[~bads]
+
+    return dat
+
+
+@functools.lru_cache(maxsize=1)
+def get_roll_pitch_tlm_safe(start, stop):
+    """Get telemetry values to compute pitch and roll in safe mode"""
```

Comment:
Might help our code search to note that these are CSS data msids

Reply:
Fixed.